packages/tendermint-rpc/src/adaptors/v0-33/requests.ts

```diff
@@ -84,6 +84,19 @@ function encodeTxSearchParams(params: requests.TxSearchParams): RpcTxSearchParam
   };
 }
 
+interface RpcValidatorsParams {
+  readonly height?: string;
+  readonly page?: string;
+  readonly per_page?: string;
+}
+function encodeValidatorsParams(params: requests.ValidatorsParams): RpcValidatorsParams {
```

Comment:
I'm a bit confused. This means we locally set a string but it is sent as a number over rpc?

Why not just use number everywhere. (The reverse, local numbers auto-encoding/decoding to rpc string, makes much more sense)

Reply:
Nevermind, it does take local numbers into rpc strings. Saw the usage below.